MIGRATION_GUIDE.md

```diff
@@ -48,5 +48,31 @@ Choose one of the following migration paths depending on your application:
     - Run `npx expo prebuild --clean` (any manual changes to Android or iOS folders will be lost)
     - Add the new callback URL to Auth0 dashboard
 - **If your project is built with Non Expo:**
+
   - To keep things as it is, set `useLegacyCallbackUrl` to true in `authorize` and `clearSession`
   - To migrate to new non-custom scheme flow, add the new callback URL to Auth0 dashboard
+  - Replace the manifest placeholders in your app's build.gradle file (typically at android/app/build.gradle) from:
```

Comment:
```suggestion
  - Change the manifest placeholders in your app's build.gradle file (typically at android/app/build.gradle):
```

Replace + from reads a bit weird. I wanted to replace it with Replace+With, or Change+From, but given the below just mentioned Old and New, I guess we can just mention `Change`.